cmd/ttn-lw-cli/commands/clients.go

```diff
@@ -94,17 +94,18 @@ var (
 		RunE: func(cmd *cobra.Command, args []string) error {
 			paths := util.SelectFieldMask(cmd.Flags(), selectClientFlags)
 
-			req := getSearchEntitiesRequest(cmd.Flags())
+			req, opt, getTotal := getSearchEntitiesRequest(cmd.Flags())
 			req.FieldMask.Paths = paths
 
 			is, err := api.Dial(ctx, config.IdentityServerGRPCAddress)
 			if err != nil {
 				return err
 			}
-			res, err := ttnpb.NewEntityRegistrySearchClient(is).SearchClients(ctx, req)
+			res, err := ttnpb.NewEntityRegistrySearchClient(is).SearchClients(ctx, req, opt)
 			if err != nil {
 				return err
 			}
+			getTotal()
```

Comment:
This looks very weird - maybe add a comment explaining why return value of this function(which returns a `uint64`) is discarded.

Also/or
```suggestion
_ = getTotal()
```
For additional clarity? Explicit `_` shows makes it obvious this is intentional and not a bug

Reply:
This is also how we do it in other places (all list commands), let's please not change all of that in this PR.

Reply:
What does this do exactly?

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/blob/master/cmd/ttn-lw-cli/commands/pagination.go#L37-L49